README.md

```diff
@@ -630,6 +632,7 @@ Transform an index against the delta. Useful for representing cursor/selection p
 #### Example
 
 ```js
-var index = 12;
-var transformedIndex = delta.transformPosition(index);
+var delta = new Delta().retain(5).insert('a');
+var transformed4 = delta.transformPosition(4); // 4
```

Comment:
Just a nitpick but since you put the result of the function in the comment, can you remove the variable declaration and assignment?